cg/store/filters/status_application_filters.py

```diff
@@ -27,6 +27,10 @@ def filter_applications_is_not_external(applications: Query, **kwargs) -> Query:
     return applications.filter(Application.is_external == False)
 
 
+def filter_applications_has_versions(applications: Query, **kwargs) -> Query:
```

Comment:
Would be nice a unit test for the filter function

Reply:
Added!